src/Rings/mpoly-ideals.jl

```diff
@@ -2072,8 +2085,40 @@ function flag_pluecker_ideal(F::Field, dimensions::Vector{Int}, n::Int)
   genz = [sum(t) for t in T]
   filter!(!iszero, genz)
   gg = unique(x ->  divexact(x, canonical_unit(x)), genz)
+  I  = ideal(gg)
 
-  return ideal(gg)
+  !minimal && return I 
+  # Computes the groebner_basis of the ideal
+  o = degrevlex(R)
+  converted_generators = collect(groebner_basis(I; ordering = o))
+  return ideal(IdealGens(R, converted_generators, o;
+                   keep_ordering=true,
+                   isReduced=true,
+                   isGB=true))
 end
 
-flag_pluecker_ideal(dimensions::Vector{Int}, n::Int) = flag_pluecker_ideal(QQ,dimensions,n)
+flag_pluecker_ideal(dimensions::Vector{Int}, n::Int; minimal::Bool=true) = flag_pluecker_ideal(QQ,dimensions,n, minimal=minimal)
+
+function flag_pluecker_ideal(ring::MPolyRing{<: FieldElem}, dimensions::Vector{Int}, n::Int; minimal::Bool=true)
+
+  L = sum(binomial(n, d) for d in dimensions)
+  @req L == ngens(ring) "The ring does not have the correct amount of variables"
+  
+  base_field = base_ring(ring)  
+  I = flag_pluecker_ideal(base_field, dimensions, n; minimal = false) 
+  
+  coeffmap = c -> begin
+    @assert is_integral(c)
+    return base_field(numerator(c))
+  end
+  h = hom(base_ring(I), ring, coeffmap, gens(ring))
```

Comment:
Do you need this `coeffmap`? `base_ring(I)` and `ring` have the same coefficient ring, so this should just work.
```suggestion
  h = hom(base_ring(I), ring, gens(ring))
```